### ISSUE TITLE
Shortwave radiation: major differences between ARPEGE and other models

### ISSUE THREAD

**Issue by patrick-zippenfenig** [open-meteo/open-meteo#881]
### Discussed in https://github.com/open-meteo/open-meteo/discussions/879

<div type='discussions-op-text'>

<sup>Originally posted by **mthiboust** June 25, 2024</sup>
I am surprised by the significative difference between ARPEGE and other models for `shortwave_radiation` data. For this parameter, the ARPEGE model is consistently 20% to 50% above the other ones (ECWMF and DWD forecasts & ERA5 reanalysis). Do you know if ARPEGE and ECWMF `ssrd` GRIB parameter is measuring the same thing defined [here](https://codes.ecmwf.int/grib/param-db/169)?

This difference is so big it cannot "just" be a prediction error! I'm sure there is a rational explanation behind this but haven't found it yet. Do you have any idea?

This plot using the historical forecast data shows this well:
![image](https://github.com/open-meteo/open-meteo/assets/14574229/7a75a825-2120-4f48-9e35-006b8324bd6b)
 </div>

**Comment by patrick-zippenfenig** [comment]
@mthiboust thanks for the information. Yes, both `ssrd` fields should use the same definition. I will investigate.

**Comment by patrick-zippenfenig** [comment]
On first sight, processing looks correct. I initially expected a processing issue, there had been many changes to the MeteoFrance download and processing chain recently.

Shortwave radiation looks a bit high from MeteoFrance models on clear sky days. As far as Open-Meteo is concerned, this data should accurately return whatever is directly output from the MeteoFrance model.

![Screenshot 2024-06-25 at 18 11 50](https://github.com/open-meteo/open-meteo/assets/11670456/de46738f-59ff-4658-a0f9-6aa796dc0a3c)

![Screenshot 2024-06-25 at 18 15 55](https://github.com/open-meteo/open-meteo/assets/11670456/2b57240a-b9dc-47b8-8f3e-9573906a46c0)

**Comment by patrick-zippenfenig** [comment]
@mthiboust After checking the solar radiation values again, I spotted the issue. I applied the conversion from Joules to Watts incorrectly. The wrong factor was `3600/10_000_000 = 0.00036`, the correct conversion factor is `1/3600 = 0.00027778`. This results in an error of 23% (`125/162 = 0.771`).

PR https://github.com/open-meteo/open-meteo/pull/885 corrects data in the API code. The fix is already applied to the API servers. Only Solar radiation from MeteoFrance was affected by this bug.

I spotted the issue, as I [just implemented](https://github.com/open-meteo/open-meteo/issues/172#issuecomment-2192052514) the KNMI Harmonie Arome models, which also provide solar radiation in Joules.

All models are now well in line. For whatever reason MeteoFrance ARPEGE World is now a bit too low, but this is just a single location.

![Screenshot 2024-06-26 at 18 34 30](https://github.com/open-meteo/open-meteo/assets/11670456/9c55888c-2982-4ba7-a548-50364c4c94f3)
